contracts/contracts/coordination/subscription/BqETHSubscription.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "./StandardSubscription.sol";
+
+/**
+ * @title BqETHSubscription
+ */
+contract BqETHSubscription is StandardSubscription {
```

Comment:
Out of curiosity: why do we need to implement a new contract for the adopter? Why not just deploy StandardSubscription with the constructor parameters set for this adopter?

Reply:
because period calculation based on immutable duration. So this update includes new duration and skipping 0 period. Otherwise it will consider 0 period with length of year instead of 6 months